src/matrix/calls/group/GroupCall.ts

```diff
@@ -200,6 +200,8 @@ export class GroupCall extends EventEmitter<{change: never}> {
         if ((this._state === GroupCallState.Joining || this._state === GroupCallState.Joined) && this.joinedData) {
             const oldMedia = this.joinedData.localMedia;
             this.joinedData.localMedia = localMedia;
+            this.localPreviewMedia?.dispose();
```

Comment:
Slightly confusing to use the public getter here for accessing and the private member for assigning, but I guess that works.